Solutions/Corvus.Tenancy.Specs/Corvus.Tenancy.Specs.csproj

```diff
@@ -52,15 +53,13 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Endjin.RecommendedPractices.GitHub" Version="2.1.12">
+    <PackageReference Include="Corvus.Testing.ReqnRoll.NUnit" Version="4.0.3" />
+    <PackageReference Include="Endjin.RecommendedPractices.GitHub" Version="2.1.15">
       <PrivateAssets>all</PrivateAssets>
       <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
     </PackageReference>
-    <PackageReference Include="FluentAssertions" Version="6.11.0" />
     <PackageReference Include="Microsoft.Extensions.Configuration.EnvironmentVariables" Version="[6.0.*,)" />
```

Comment:
When we've been upgrading other project to .NET 8.0, we've also fixed all `Microsoft.Extensions.*` extensions to version `[8.*,)`

Reply:
That should be pointing to the `Microsoft.Extensions.Configuration.EnvironmentVariables` entry!